apps/style/netsim/style.scss

```diff
@@ -776,5 +776,20 @@ $wide-column-width: 121px;
       padding-right: 14px;
     }
   }
+
+  // Really garish terrible style for debug buttons that we should not ship.
```

Comment:
These styles are fine for this purpose. But, there are some debug controls which hover in the bottom right like the ones visible on this level (if logged in as admin) which you could copy: http://levelbuilder.studio.code.org/s/artist/stage/1/puzzle/10 
![screen shot 2015-04-29 at 2 50 08 pm](https://cloud.githubusercontent.com/assets/8001765/7402239/20012704-ee7f-11e4-980d-bb754bac9d34.png)